### PR TITLE
Tweak: Add the option to disable header & footer [ED-12744]

### PR DESCRIPTION
**Pull request by rami-elementor**

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -126,7 +126,9 @@ function hello_elementor_scripts_styles() {
 				[],
 				HELLO_ELEMENTOR_VERSION
 			);
+		}
 
+		if ( apply_filters( 'hello_elementor_header_footer', true ) ) {
```
Comment (rami-elementor) [inline functions.php:131]: Don't load header/footer CSS.

File: footer.php [hunk 2]
```diff
@@ -12,10 +12,12 @@
 }
 
 if ( ! function_exists( 'elementor_theme_do_location' ) || ! elementor_theme_do_location( 'footer' ) ) {
-	if ( did_action( 'elementor/loaded' ) && hello_header_footer_experiment_active() ) {
-		get_template_part( 'template-parts/dynamic-footer' );
-	} else {
-		get_template_part( 'template-parts/footer' );
+	if ( apply_filters( 'hello_elementor_header_footer', true ) ) {
```
Comment (KingYes) [inline footer.php:15]: use `not` condition and exit from the function

File: header.php [hunk 3]
```diff
@@ -33,9 +33,11 @@
 
 <?php
 if ( ! function_exists( 'elementor_theme_do_location' ) || ! elementor_theme_do_location( 'header' ) ) {
-	if ( did_action( 'elementor/loaded' ) && hello_header_footer_experiment_active() ) {
-		get_template_part( 'template-parts/dynamic-header' );
-	} else {
-		get_template_part( 'template-parts/header' );
+	if ( apply_filters( 'hello_elementor_header_footer', true ) ) {
```
Comment (KingYes) [inline header.php:36]: use `not (!)` condition and exit from the function

**Review by KingYes — CHANGES_REQUESTED** [review]

File: header.php (hunk 3 above)
Comment (rami-elementor) [reply header.php:36]: It's not a function...

**Comment by rami-elementor** [comment]
@KingYes, just as we discussed, I've moved the header & footer filter hook to a separate function.

**Review by KingYes — APPROVED** [review]

**Pull request merged**